packages/tc-markdown-parser/lib/tree-mutators/apply-blacklist.js

```diff
@@ -0,0 +1,26 @@
+const defaultBlacklistProperties = [
+	'lastReleaseTimestamp',
+	'dependentCapabilities',
+	'dependentProducts',
+	'dependents',
+	'lastServiceReviewDate',
+	'lastSOSReport',
+	'piiSources',
+	'recursiveDependencies',
+	'recursiveDependentProducts',
+	'recursiveDependents',
+	'replacedBy',
+	'repositories',
+	'SF_ID',
+	'sosTrafficLight',
+	'stakeholders',
+	'updatesData',
+	'dataOwner',
+	'gdprRetentionProcess',
+	'gdprErasureProcess',
+];
+
+const applyBlacklistProperties = (additionalBlacklistProperties = []) =>
+	defaultBlacklistProperties.concat(additionalBlacklistProperties);
```

Comment:
This doesn't need default blacklist. Should default to [] if none provided by user (runbook.md should take care of defining the blacklist we've been using so far)

Reply:
Ah, sorry I forgot to remove this